baseapp/grpcserver.go

```diff
@@ -23,7 +24,7 @@ import (
 func (app *BaseApp) GRPCQueryRouter() *GRPCQueryRouter { return app.grpcQueryRouter }
 
 // RegisterGRPCServer registers gRPC services directly with the gRPC server.
-func (app *BaseApp) RegisterGRPCServer(server gogogrpc.Server) {
+func (app *BaseApp) RegisterGRPCServer(server gogogrpc.Server, logQueries bool) {
```

Comment:
Can you add an API breaking changelog?

Reply:
Added here https://github.com/cosmos/cosmos-sdk/pull/19049/commits/e82a08462856697bf3979e513ea8815309303642